plottr/gui/widgets.py

```diff
@@ -88,7 +88,7 @@ def __init__(self, parent: Optional[QtWidgets.QMainWindow] = None,
         if fc is not None:
             self.addNodeWidgetsFromFlowchart(fc, **kw)
 
-        self.setDefaultStyle()
+        #self.setDefaultStyle()
```

Comment:
Rather than commenting this out we should update setDefaultStyle to set the font size to 10*devicePixelRatio